yarn-project/noir-libs/noir-aztec/src/abi.nr

```diff
@@ -277,6 +281,10 @@ impl ContractStorageUpdateRequest {
     fn hash(self) -> Field {
         dep::std::hash::pedersen_with_separator(self.serialize(), GENERATOR_INDEX__PUBLIC_DATA_UPDATE_REQUEST)[0]
     }
+
+    fn empty() -> Self {
+        Self { storage_slot: 0, old_value: 0, new_value: 0 }
+    }
 }
 
 fn empty_contract_storage_update_request() -> ContractStorageUpdateRequest {
```

Comment:
My brain want us to use `ContractStorageUpdateRequest::empty()` where this was used and get rid of this one.